internal/weapons/catalyst/hakushin/hakushin.go

```diff
@@ -83,5 +83,6 @@ func NewWeapon(c *core.Core, char *character.CharWrapper, p weapon.WeaponProfile
 	c.Events.Subscribe(event.OnSuperconduct, hrfunc(attributes.Cryo, "hr-sc"), fmt.Sprintf("hakushin-ring-%v", char.Base.Key.String()))
 	c.Events.Subscribe(event.OnQuicken, hrfunc(attributes.Dendro, "hr-quick"), fmt.Sprintf("hakushin-ring-%v", char.Base.Key.String()))
 	c.Events.Subscribe(event.OnAggravate, hrfunc(attributes.Dendro, "hr-agg"), fmt.Sprintf("hakushin-ring-%v", char.Base.Key.String()))
+	c.Events.Subscribe(event.OnHyperbloom, hrfunc(attributes.Dendro, "hr-hyperbloom"), fmt.Sprintf("hakushin-ring-%v", char.Base.Key.String()))
```

Comment:
Do we also need the damage bonus for hydro?

Reply:
nope